.github/workflows/pylint.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install nomad-lab==1.1.1 --extra-index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        pip install astroid==2.5.1
```

Comment:
It is a requirement form nomad-lab and is in its requirements in this version. However, a newer version gets installed in the action, which has some breaking API changes, which lets the nomad pylint plugin fail. It was just a quick fix here, because the issue needs to be resolved in nomad and not in nexus-parser.

Reply:
You can see the error in the failing pipeline in master.

Reply:
LGTM